src/mesh/NodeDB.cpp

```diff
@@ -846,6 +855,12 @@ void NodeDB::loadFromDisk()
         }
     }
 
+    state = loadProto(uiconfigFileName, meshtastic_DeviceUIConfig_size, sizeof(meshtastic_DeviceUIConfig),
```

Comment:
Does this need to be wrapped in some kind of exclude, for nodes not using the new UI?

Reply:
No. We don't know at compile time if the node has connected a UI or not.

Reply:
Thanks!